### PR TITLE
Brooklyn Feature:- Custom routes set on Gateway.

### PR DESCRIPTION
**Pull request by Nilambari**
## Description

These are command lets changes for setting custom routes specified by customer on Virtual network gateway. This can be done while creating new gateway or updating an existing gateway.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by Nilambari** [comment]
PowerShell cmdlet review is approved :- Azure/azure-powershell-cmdlet-review-pr#285

**Comment by Nilambari** [comment]
@markcowl @MikhailTryakhov Can you Please take a look and help approve? Thanks!

**Comment by MiYanni** [comment]
@Nilambari Merge conflicts.

File: src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs [hunk 1]
```diff
@@ -127,11 +133,6 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             ValueFromPipelineByPropertyName = true,
             ParameterSetName = VirtualNetworkGatewayParameterSets.RadiusServerConfiguration,
             HelpMessage = "P2S External Radius server address.")]
-        [Parameter(
-            Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            ParameterSetName = VirtualNetworkGatewayParameterSets.RadiusServerConfiguration + VirtualNetworkGatewayParameterSets.UpdateResourceWithTags,
-            HelpMessage = "P2S External Radius server address.")]
```
Comment (MiYanni) [inline src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs]: Why are you deleting this parameter set?

File: src/Network/Network/help/Set-AzVirtualNetworkGateway.md [hunk 2]
```diff
@@ -167,6 +168,19 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -CustomRoute
+Custom routes AddressPool specified by customer```yaml
```
Comment (MiYanni) [inline src/Network/Network/help/Set-AzVirtualNetworkGateway.md]: Doc gen bug. Please move 
```
```yaml
```
down two lines (same way as other parameters).

File: src/Network/Network/help/New-AzVirtualNetworkGateway.md [hunk 3]
```diff
@@ -137,6 +138,19 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -CustomRoute
+Custom routes AddressPool specified by customer```yaml
```
Comment (MiYanni) [inline src/Network/Network/help/New-AzVirtualNetworkGateway.md]: Doc gen bug. Please move 
```
```yaml
```
down two lines (same way as other parameters).

File: src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs [hunk 4]
```diff
@@ -140,14 +141,15 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             ValueFromPipelineByPropertyName = true,
             ParameterSetName = VirtualNetworkGatewayParameterSets.RadiusServerConfiguration,
             HelpMessage = "P2S External Radius server secret.")]
-        [Parameter(
-            Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            ParameterSetName = VirtualNetworkGatewayParameterSets.RadiusServerConfiguration + VirtualNetworkGatewayParameterSets.UpdateResourceWithTags,
-            HelpMessage = "P2S External Radius server secret.")]
```
Comment (MiYanni) [inline src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs]: Why are you deleting this parameter set?

**Comment by Nilambari** [comment]
@MiYanni Fixed merge conflics.

File: src/Network/Network/help/Set-AzVirtualNetworkGateway.md (hunk 2 above)
Comment (Nilambari) [reply src/Network/Network/help/Set-AzVirtualNetworkGateway.md]: Fixed.

File: src/Network/Network/help/New-AzVirtualNetworkGateway.md (hunk 3 above)
Comment (Nilambari) [reply src/Network/Network/help/New-AzVirtualNetworkGateway.md]: Fixed.

**Comment by Nilambari** [comment]
Fixed help files.

**Pull request merged**